.github/workflows/test.yml

```diff
@@ -6,33 +6,43 @@ on:
       environment:
         type: string
```

Comment:
- this can be improved by using `type: environment` => this will then show a dropdown box to select one of the existing environments